.infra/.nvmrc

```diff
@@ -1 +1 @@
-18
+20
```

Comment:
shoudl we do same as one in api? `20.12`

Reply:
Set it to 20.12